### ISSUE TITLE
A11y_Azure Data Studio_Query Plan (Preview)_ScreenReader: NVDA is not announcing tooltip present at any node.

### ISSUE THREAD

**Issue by Rama8345** [microsoft/azuredatastudio#20765]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #SQL Azure Data Studio; #SH-SQL Azure Data Studio-Win32-OCT2022; #DesktopApp; #ScreenReader; #WCAG1.3.1; #FTP; #NVDA; #A11YCAIOH; #Regressed:11-02-22; #Closed

### Environment Details:
Application Name: Azure Data Studio
Windows Version: Windows 11

### Repro Steps:
1. Launch the "Azure data studio" application.
2. Connect with the provided SQL server.
3. TAB to "New Query" button and hit ENTER key.
4. Enter the SQL query and TAB to "Estimated Plan" button and hit ENTER key.
5. "Query Plan (Preview) screen appear.
6. Observe that NVDA is not announcing tooltip present at any node.

### Actual:
NVDA is not announcing tooltip present at any node.

### Expected Result:
NVDA should announce tooltip present at any node.

### Attachment:
![A11y_Azure Data Studio_Query Plan (Preview)_ScreenReader](https://user-images.githubusercontent.com/115216656/194811884-34b25c06-df94-4420-9bc1-5729a6ccfa09.gif)

**Comment by aasimkhan30** [comment]
@Rama8345, this issue is fixed in the lates insiders. Now, when you click on the node the tooltip is shown and announced. Can you please verify and close? Thanks!

**Comment by Rama8345** [comment]
@aasimkhan30 
Verified the bug in the latest insiders build, but we are getting below error.
![image](https://user-images.githubusercontent.com/115216656/199224409-66c9dff7-11c4-4f6a-8c1c-657559e55fc1.png)

**Comment by Rama8345** [comment]
Verified the issue in the latest insiders build. The issue is fixed. So closing the bug. Please find attachment for reference.
![20765_Closed](https://user-images.githubusercontent.com/115216656/199409770-99d2c2ec-4d51-42aa-8760-b69f89713cb1.gif)
